changelog/upgrade-notes.md

```diff
@@ -33,6 +33,10 @@ Users compiling their code with the library should note that the following typed
 * ol.style.RegularShapeRenderOptions to ol.RegularShapeRenderOptions
 * ol.style.StyleFunction to ol.StyleFunction
 
+#### KML format now uses URL()
+
+The URL constructor is supported by all modern browsers, but not by older ones, such as IE. To use the KML format in such older browsers, a URL polyfill will have to be loaded before use.
```

Comment:
I'll check that out. The tests are a better fit for URL(), as they're testing all the pieces of the uri, so we can keep the polyfill there.